doc/eventhandler/tokenhandler.rst

```diff
@@ -169,6 +169,16 @@ Using the action ``set failcounter`` you can reset the fail counter by
 setting it to 0 or also "block" the token by setting the fail counter to what
 ever value the "max_fail" is, e.g. 10. Only integer values are allowed.
 
+change failcounter
```

Comment:
honestly, I find this naming a bit difficult.

*set failcounter* is also a *change*.